iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractCurrentLimitsTest.java

```diff
@@ -678,4 +679,21 @@ public void testAdderSetPermanentLimitWithInfiniteDurationValue() {
         assertEquals(1, names.size());
         assertTrue(names.contains("TL1"));
     }
+
+    @Test
+    public void testAdderWithValueZero() {
+        Line line = createNetwork().getLine("L");
+        CurrentLimitsAdder adder = line.newCurrentLimits1()
+                .setPermanentLimit(0)
+                .beginTemporaryLimit()
+                    .setName("TEST")
+                    .setAcceptableDuration(Integer.MAX_VALUE)
+                    .setValue(0)
+                .endTemporaryLimit();
+        adder.add();
+        Optional<CurrentLimits> optionalLimits = line.getCurrentLimits(TwoSides.ONE);
+        assertTrue(optionalLimits.isPresent());
+        CurrentLimits limits = optionalLimits.get();
+        assertEquals(0, limits.getTemporaryLimit(Integer.MAX_VALUE).getValue());
```

Comment:
In your second commit, you have removed the assertion on the permanent limit's value.
Could you restore it?